src/planscape/impacts/models.py

```diff
@@ -232,9 +243,13 @@ class TreatmentPrescription(
         related_name="tx_prescriptions",
         on_delete=models.SET_NULL,
         null=True,
+        help_text="Stand which Treatment Prescription will be applied.",
     )
 
-    geometry = models.PolygonField(srid=settings.CRS_INTERNAL_REPRESENTATION)
+    geometry = models.PolygonField(
+        srid=settings.CRS_INTERNAL_REPRESENTATION,
+        help_text="Geomeotry of the Tratment Prescription.",
```

Comment:
just some small typos here:
Geometry of the Treatment Prescription.

Reply:
Thank you. I fixed it.